Bio/Blast/NCBIWWW.py

```diff
@@ -131,6 +137,12 @@ def qblast(
                       manually set parameters like word size and e value. Turns
                       off when sequence length is > 30 residues. Default: None.
      - service        plain, psi, phi, rpsblast, megablast (lower case)
+     - organisms      A dictionary that defines the organisms that will be
+                      included/excluded in the search. The key is the name
+                      of the organism, following the taxonomy convention
+                      ie. "Bacteria (taxid:2)" and the value is a boolean
```

Comment:
ie -> eg

I assume (please confirm) this works with any taxonomy node like "Bacteria" and not just leaf nodes (like a specific species)?

Reply:
On my use cases, it is indeed the case. And it does set difference between domains, eg.

`
{
  "Bacteria (taxid:2)": False,
  "E. coli (taxid:562)": True
}
`

Will give you results that are bacteria but not "E. coli". This particular behavior is what I am relying on for my use case.